docs/ar/week06/06-2.md

```diff
@@ -0,0 +1,306 @@
+lang-ref: ch.06-2
+lecturer: Yann LeCun
+title: RNNs, GRUs, LSTMs, Attention, Seq2Seq, and Memory Networks
+authors: Jiayao Liu, Jialing Xu, Zhengyang Bian, Christina Dominguez
+date: 2 March 2020
```

Comment:
What happened here? There is no header, the title is not translated, missing many fields.
Please, update.